recipe/recipe-scripts-license.txt

```diff
@@ -0,0 +1,27 @@
+BSD-3-Clause license
```

Comment:
did the bot add this? haven't seen it before, and it isn't referenced in the `license_file` section... and we _probably_ don't need it.

Reply:
Yeah, it appeared after this last one, and i assumed it was something I had to commit.

Reply:
happy to remove though

Reply:
yeah, go ahead and remove it, push, and then fire up the bot command to rerender

Reply:
lol, I think I must have captured some intermediate state of something because it had been automatically marked for deletion on my local branch.  🤯